server/graphql/types/src/types/program/rnr_form_line.rs

```diff
@@ -76,6 +77,10 @@ impl RnRFormLineNode {
         self.rnr_form_line_row.calculated_requested_quantity
     }
 
+    pub async fn low_stock(&self) -> LowStockStatus {
```

Comment:
In hindsight... should this have just been calculated and the correct enum returned here? Rather than storing on the line?